### DISCUSSION TITLE
trigger before the power off

### DISCUSSION THREAD

**Discussion by fachul1234** [arendst/Tasmota#12185] [Q&A] [answered]
hi guys 

i have a dehumidifier which have evaporator drying process before it turned off.
in other word, before it turned off it should be alive for 30sec to do drying process.

i got sonoff basic with additional relay module.
so that i can control main power and dehumidifier's on/off touch switch.

but A single relay should be controlled 2 different way to emulate the machine perfectly .
on : relay activate for 1sec to emulate machine "on" after power on.
off : relay activate for 1sec, 30sec before power off to maintain machine's drying process.

how can i solve this matter?
i'm pretty noob in this field let me have some wisdom!!

**Comment by sfromis** [comment, marked as answer]
You can setup PulseTime to have the relay activate a pulse instead of fixed "on" state.
